### DISCUSSION TITLE
grouping in CSS3

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#52] [Q&A] [answered]
What is the use of grouping in CSS3?

**Comment by Bunny77K9** [comment, marked as answer]
Grouping is used in CSS3 to give users the ability to reuse and apply the same CSS style element to multiple HTML entities, using just one single declaration statement.

A simple example of grouping is as shown below:

#grouped g, ul { padding-top: 20px; margin: 1; }
